core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/config/AbstractFlowConfiguration.java

```diff
@@ -266,22 +270,23 @@ public void addTransition(S from, S to, E with, S fail, E withFailure) {
                 builder.addTransition(from, to, with, failure);
                 return builder;
             }
+
+            public Builder<S, E> noFailureEvent() {
+                builder.addTransition(from, to, with, failure, null);
```

Comment:
noFailureEvent the method name, and we pass failure to the transition. Annoying for me.